src/components/layout/Space.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react'
```

Comment:
Due to now known performance impact of additional `View` components, I would push back against the architecture of the `Space` component and to use padding on existing components instead where possible.